debugAdapterProtocol.json

```diff
@@ -3397,6 +3417,12 @@
 			"required": [ "verified" ]
 		},
 
+		"SteppingGranularity": {
+			"type": "string",
+			"description": "The granularity of the requested step.",
+			"enum": [ "statement", "line", "instruction" ]
```

Comment:
Please add descriptions for the individual values (by means of an "enumDescriptions" array).

Reply:
Added. 

I noticed that the comments weren't getting added to specification.md so I also went ahead and updated the generator to add comments for standalone enum definitions. It added comments on a couple of other places as well. I can also revert those changes if you'd prefer to leave it as is for now (or to handle it separately).

Reply:
Reverted for now - was not quite as simple as I initially thought.